### ISSUE TITLE
rkChainLinkZeroAcc() is now available.

### ISSUE THREAD

**Issue by zhidao** [mi-lib/roki#12]
@categoryik rkChainLinkZeroAcc() has been newly defined in rk_chain.c. It computes the acceleration of a point on a link and the angular acceleration of the same link with respect to the inertia frame at zero-joint acceleration. It is equivalent with J dot q dot. See test/jacobi_test.c for how to use it.

**Comment by categoryik** [comment]
Thank you for your work.
 I have understood the name come from following equation if $\ddot{\mathbf{q}}=\mathbf{0}$.
$$\ddot{\mathbf{p}}=\dot{\mathbf{J}}\ddot{\mathbf{q}}+\dot{\mathbf{J}}\dot{\mathbf{q}}$$

**Comment by zhidao** [comment]
Right:)

**Comment by categoryik** [comment]
We found that `rkChainLinkZeroAcc()` includes Gravity effects because `rkChainUpdateRate()` is used in `rkChainLinkZeroAcc()`. it should be `rkchainUpdateRateUnderZeroGravity()`.

**Comment by zhidao** [comment]
Thank you for your pointing out. As I replied to the pull request from you, I implemented rkChainLinkZeroAcc0G() in another way. The idea is to implement rkChainLinkZeroAccG() to accept an acceleration of field, which can be either RK_GRAVITY6D or ZVEC6DZERO, and added rkChainLinkZeroAcc() and rkChainLinkZeroAcc0G() as macros.
The same method has been used for rkChainUpdateRate(), rkChainUpdateID(), and rkChainID() families.
